doc/doc_ch/FAQ.md

```diff
@@ -170,10 +170,6 @@ A：可以在预测时调小 det_db_box_thresh 阈值，默认为0.5, 可调小
 
 **A**：opencv的tps需要标出上下边界对应的点，这个点很难通过传统方法或者深度学习方法获取。PaddleOCR里StarNet网络中的tps模块实现了自动学点，自动校正，可以直接尝试这个。
 
-#### Q: 如何识别招牌或者广告图中的艺术字？
-
-A: 招牌或者广告图中的艺术字是文本识别一个非常有挑战性的难题，因为艺术字中的单字和印刷体相比，变化非常大。如果需要识别的艺术字是在一个词典列表内，可以将改每个词典认为是一个待识别图像模板，通过通用图像检索识别系统解决识别问题。可以尝试使用PaddleClas的图像识别系统。
-
```

Comment:
I think it would be better to delete the following paragraph

```
Q: 如何识别招牌或者广告图中的艺术字？
A: 招牌或者广告图中的艺术字是文本识别一个非常有挑战性的难题，因为艺术字中的单字和印刷体相比，变化非常大。如果需要识别的艺术字是在一个词典列表内，可以将该每个词典认为是一个待识别图像模板，通过通用图像检索识别系统解决识别问题。可以尝试使用PaddleClas的图像识别系统PP-shituV2。
```

Reply:
Should the second be deleted and the first retained?

Reply:
@xu8117, yes.

Reply:
@GreatV I'm done with the doc. Anything else I need to change. And git commit log is a bit messy, do I need to re-commit a pr

Reply:
@xu8117, no need re-commit.